### PR TITLE
Feature/cards

### PR DESCRIPTION
**Pull request by ammiratafabiano**
implementata pagina filtri e filtro distanza

### CODE REVIEW

**Comment by calca** [comment]
ERROR in Type FilterPage in /home/vsts/work/1/s/mobile/src/app/modules/filter/filter.page.ts is part of the declarations of 2 modules: FilterProfilePageModule in /home/vsts/work/1/s/mobile/src/app/modules/filter-profile/filter-profile.module.ts and FilterPageModule in /home/vsts/work/1/s/mobile/src/app/

**Comment by calca** [comment]
effettua la build con ionic build --prod

File: mobile/package.json [hunk 1]
```diff
@@ -68,7 +70,8 @@
   "description": "An Ionic project",
   "cordova": {
     "plugins": {
-      "cordova-plugin-nativegeocoder": {}
+      "cordova-plugin-nativegeocoder": {},
+      "cordova-plugin-geolocation": {}
```
Comment (citerad) [inline mobile/package.json]: togliamo "cordova-plugin-geolocation

File: mobile/src/app/app.module.ts [hunk 2]
```diff
@@ -17,6 +17,7 @@ import { TranslateHttpLoader } from '@ngx-translate/http-loader';
 import { TranslateLoader, TranslateModule } from '@ngx-translate/core';
 import { NativeGeocoder } from '@ionic-native/native-geocoder/ngx';
 import { AgmCoreModule } from '@agm/core';
+import { Geolocation } from '@ionic-native/geolocation/ngx';
```
Comment (citerad) [inline mobile/src/app/app.module.ts]: https://capacitor.ionicframework.com/docs/apis/geolocation/

File: mobile/package.json (hunk 1 above)
Comment (ammiratafabiano) [reply mobile/package.json]: ho ricevuto durante il merge questo plugin, che devo fare?

**Review by citerad — APPROVED** [review]

**Pull request merged**